### ISSUE TITLE
Error message when trying to invest in other people's funds

### ISSUE THREAD

**Issue by Mylittlerapture** [melonproject/melon-lab#329]
I was trying to invest in Fund that wasn't mine several times with reasanable amount of Gas paid and definitly enough money on the wallet to cover operation costs. Yet I had this message pop up each time.

GraphQL error: Error in prepare transaction Participation@0x0B965e385A72335c761DA3f59A37BC264A43cf39.requestInvestment("26363636363636363","29000000000000000","0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2")): Gas estimation (preflight) failed for requestInvestment("26363636363636363", "29000000000000000", "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2"): Returned error: invalid argument 0: json: cannot unmarshal hex string without 0x prefix into Go value of type *hexutil.Big

**Comment by jennazenk** [comment]
Maybe the fund is not open ended ie. has a whitelist of authorized investors? You can see on the fund's risk management profile if that is the case. If not, please let us know.

**Comment by Mylittlerapture** [comment]
@jennazenk No, this wasn't an issue. I was able to initiate investment yesterday. I can't say for sure what was different about it, except that I invested slightly bigger amount this time. My investment didn't come through though, because somehow I missed time windows for the second step of the investment process. When I opened Interface today and tried to confirm my investment the was a message "Your request is expired!". Here is actual transaction of investment https://etherscan.io/tx/0xee1f6d07af47ebee59a58eeb44cf26a3b5f31b13e450fb6007ef456861648743 that happend about a day ago at this moment. So I have no idea how can I missed executable period if is supposed to last for 24 hours.
